### DISCUSSION TITLE
Does this script also works for migrating RHEL* systems?

### DISCUSSION THREAD

**Discussion by azamsab** [oracle/centos2ol#87] [Q&A]
Hi All,

I'd like to know if we can also use this script (centos2ol.sh) to migrate RHEL* systems to OEL* ?
Or do we have another script to convert RHEL* systems to OEL?

Any advise will be highly appreciated. Thanks in advance!

**Comment by honglinsu** [comment]
centos2ol script is for CentOS, not for RHEL instances. 

To install Oracle Linux updates for RHEL instances, follow the instructions: http://yum.oracle.com/getting-started.html#red-hat-enterprise-linux-centos

We recommend Oracle Linux consulting services to ensure a smooth transition from Red Hat support to Oracle Linux support.

**Comment by arthururbano** [comment]
any chance of having a script to convert from Fedora 41 to OEL?
